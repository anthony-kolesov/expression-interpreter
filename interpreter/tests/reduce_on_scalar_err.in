out reduce(9, 0, q t -> q + t + 1)
