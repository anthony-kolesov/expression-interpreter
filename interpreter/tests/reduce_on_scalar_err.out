ERROR:1:Can't perform reduce operation on scalar value.
